nexus/db-queries/src/db/pool.rs

```diff
@@ -140,4 +151,36 @@ impl Pool {
     ) -> anyhow::Result<qorb::claim::Handle<QorbConnection>> {
         Ok(self.inner.claim().await?)
     }
+
+    /// Stops the qorb background tasks, and causes all future claims to fail
+    pub async fn terminate(&self) {
+        let _termination_result = self.inner.terminate().await;
+        self.terminated.store(true, std::sync::atomic::Ordering::SeqCst);
+    }
+}
```

Comment:
What happens if terminate is called more than once?

Reply:
It would return an error, but it's otherwise idempotent.

See: https://docs.rs/qorb/latest/src/qorb/pool.rs.html#486-494